File: comms/bulk-messaging/senders/fetch-sender/fetch-sender.php
<?php

namespace Example;

use Twilio\TwilioComms;

$client = new TwilioComms(
    accountId: '<username>',
    authToken: '<password>',
);
$client->senders->fetch(
    'comms_sender_01h9krwprkeee8fzqspvwy6nq8',
);

File: comms/bulk-messaging/senders/fetch-sender/fetch-sender.py
from twilio import TwilioComms

client = TwilioComms(
    account_id="<username>",
    auth_token="<password>",
)

client.senders.fetch(
    sender_id="comms_sender_01h9krwprkeee8fzqspvwy6nq8",
)
